detection-rules/text_callback_scam_unsolicited.yml

```
@@ -0,0 +1,30 @@
+name: "Callback scam with no attachment (first-time sender)"
```

we're not actually checking if there's no attachment, an alternative might be:

```suggestion
name: "Callback scam in message body (first-time sender)"
```